RevenueCatUI/Templates/LinTemplates/LinTemplateStep1View.swift

```diff
@@ -9,10 +9,45 @@ import Foundation
 import RevenueCat
 import SwiftUI
 
+struct LinTemplateStep1Configuration: Decodable {
+    enum CodingKeys: String, CodingKey {
+        case titleKey = "title_key"
+        case imageName = "image_name"
+    }
+    
+    let titleKey: String?
+    let imageName: String
+}
+
+extension LinTemplateStep1Configuration {
+    static let `default` = LinTemplateStep1Configuration(
+        titleKey: nil,
+        imageName: "paywall-first-step-general"
```

Comment:
so the plan is to configure the bundle in a different way - so that iOS/mac distinction is given to us right?

Reply:
The images will be the same for both platforms now.